server/src/main/java/com/objectcomputing/checkins/services/memberprofile/MemberProfileServices.java

```diff
@@ -1,12 +1,12 @@
 package com.objectcomputing.checkins.services.memberprofile;
 
-import java.util.Set;
+import java.util.List;
 import java.util.UUID;
 
 public interface MemberProfileServices {
     MemberProfile getById(UUID id);
 
-    Set<MemberProfile> findByValues(String name, String role, UUID pdlId);
+    List<MemberProfile> findByValues(String name, String role, UUID pdlId);
```

Comment:
Why a list, wouldn't a set be more apropos since no duplicates? Doesn't matter either way we decide.